reports/templates/reports/admin/event_enrolments.html

```diff
@@ -44,8 +44,7 @@
             </td>
             <td>{{enrolment.study_group.group_size}}</td>
             <td>{{enrolment.study_group.amount_of_adult}}</td>
-            <td>{{enrolment.study_group.extra_needs}}</td>
```

Comment:
should the extra needs column be removed?? I guess not, but if it should, the title should be removed too.

Reply:
Well spotted sir! Fixed.